### ISSUE TITLE
Release current master HEAD to fix bundler conflicts in Rails 5.2?

### ISSUE THREAD

**Issue by nunosilva800** [activeadmin/inherited_resources#442]
While adding ActiveAdmin to Rails 5.2.beta2, the following bundle conflict happens:

```
Bundler could not find compatible versions for gem "actionpack":
  In snapshot (Gemfile.lock):
    actionpack (= 5.2.0.beta2)

  In Gemfile:
    activeadmin (~> 1.1.0) was resolved to 1.1.0, which depends on
      inherited_resources (~> 1.7) was resolved to 1.7.1, which depends on
        actionpack (< 5.1, >= 3.2)

    rails (~> 5.2.0.beta2) was resolved to 5.2.0.beta2, which depends on
      actionpack (= 5.2.0.beta2)

    rspec-rails was resolved to 3.7.2, which depends on
      actionpack (>= 3.0)

    rails (~> 5.2.0.beta2) was resolved to 5.2.0.beta2, which depends on
      sprockets-rails (>= 2.0.0) was resolved to 3.2.1, which depends on
        actionpack (>= 4.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

```

This is fixed by specifying:

```
gem 'inherited_resources',
    github: 'activeadmin/inherited_resources',
    ref: '46493ce10fc7d9f0bcd753f5d298c67ca83cf9b0'
gem 'activeadmin', '~> 1.1.0'
```

So simply releasing current HEAD will allow bundler to avoid this conflict.

-----

ps: there's no 1.7.2 tag :(

**Comment by varyonic** [comment]
See #443. I'm using
```
gem 'inherited_resources', github: 'activeadmin/inherited_resources', branch: 'rails-5.2'
```
1.7.2 tag: https://github.com/activeadmin/inherited_resources/tree/1.7.2

**Comment by rafaelfranca** [comment]
Can we keep the same pattern for all tags? We use v1.7.1. `rake release` take care of creating the tag automatically with the right pattern. I just created `v1.7.2`

**Comment by varyonic** [comment]
My guess is that @fivell did that with GitHub release.  GitHub has its own default behavior.

**Comment by varyonic** [comment]
Resolved by #443
